desktop/src/main/java/bisq/desktop/primary/main/content/wallet/send/WalletSendView.java

```diff
@@ -37,6 +38,10 @@ public WalletSendView(WalletSendModel model, WalletSendController controller) {
         TextField amountTextField = createTextField(Res.get("amount") + ":");
         amountTextField.textProperty().bindBidirectional(model.amountProperty());
 
+        PasswordField passphraseTextField = new PasswordField();
+        passphraseTextField.setPromptText(Res.get("passphrase") + ":");
+        passphraseTextField.textProperty().bindBidirectional(model.passphraseProperty());
```

Comment:
Bindings should be set in the `onViewAttached` method and unbind called at `onViewDetached` to avoid potential memory leaks. Most of the time it would not cause a leak as the reference resource gets removed as well but there might be edge cases where its not the case and then its very hard to find if we dont follow that more strict pattern.